packages/core/src/config/load.ts

```diff
@@ -61,6 +49,16 @@ export async function loadConfig(configPath?: string, customExtends?: string[]):
   );
 }
 
+export async function getConfig(configPath: string | undefined = findConfig()) {
+  if (!configPath) return {};
+  try {
+    const rawConfig = (await loadYaml(configPath)) as RawConfig;
+    return transformConfig(rawConfig);
+  } catch (e) {
+    throw new Error(`Error parsing config file at '${configPath}': ${e.message}`);
+  }
+}
+
 function findConfig() {
```

Comment:
`redocly.yaml` and `redocly.yml` support is missing here

Reply:
This should work fine:
```
 const supportedConfigNames = ['redocly.yaml', 'redocly.yml', '.redocly.yaml', `.redocly.yml`];
  const configName = supportedConfigNames.find((fileName) => {
    return existsSync(`${definitionDir}/${fileName}`);
  });
  return configName;
  ```

Reply:
There was this separate PR about that I think: https://github.com/Redocly/openapi-cli/pull/557/files